FruitFruit/FruitFruit/Controllers/CheckOrderViewController.swift

```diff
@@ -10,21 +10,26 @@ import Lottie
 
 class CheckOrderViewController: UIViewController, UIGestureRecognizerDelegate {
     
-    //TODO: 데이터 바인딩
     var fruitOrder = FruitOrder(saleFruitId: "fruitUserId", name: "여름오렌지", dueDate: Date(), amount: 3, price: 800, status: "Checking", user: Storage().fruitUser!, place: "포스텍 C5", time: 13)
     
     let animationView = AnimationView()
+    var isOrderCompleted: Bool = false
     
     @IBOutlet weak var backgroundView: UIView!
     @IBOutlet weak var titleLabel: UILabel!
     @IBOutlet weak var secondaryTitleLabel: UILabel!
     @IBOutlet weak var orderButton: UIButton!
     
     @IBAction func onOrderButtonClicked(_ sender: UIButton) {
-        playLottie()
-        addOrder(fruitOrder: fruitOrder)
-        DispatchQueue.main.asyncAfter(deadline: DispatchTime.now() + 3) {
-            self.navigateToOrderResultView()
+        if isOrderCompleted == true {
+            orderButton.isUserInteractionEnabled = false
+        } else {
+            isOrderCompleted = true
```

Comment:
```suggestion
            orderButton.isUserInteractionEnabled ? 
```

isOrderCompleted 플래그 비트의 역할 -> orderButton 내부의 isUserInteractionEnabled 비트로 줄 수 있을 것 같습니다.

viewDidLoad에서 orderButton을 먼저 true로 초기화 -> true인 상태에서만 처음 한 번 false로 주기 + 로티 이미지 및 다음 네비게이션으로 줄 수도 있을 것 같아요~~

Reply:
감사합니다!